src/src/components/form/CustomSelectInput.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <div :class="selectContainerClasses">
+    <label
+      v-if="undefinedCheck(select.prefix)"
+      class="select-prefix"
+      v-html="undefinedCheck(select?.prefix)"
+    />
+    <fa
+      v-if="select.fieldIcon"
+      :icon="fieldIconChoice"
+      class="select-icon" />
+    <label
+      :for="select.fieldName"
+      class="select-label">
+      <select
+        :name="select.fieldName"
+        :id="id"
+        :disabled="select.disabled"
+        :value="selectedValueState"
+        :class="customSelectClasses"
+        @change="$emit('selectValueChanged', selectedValueState)"
+      >
+          <option
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 8 spaces but found 10 spaces.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
        <option
```